risc0/cargo-risczero/Cargo.toml

```diff
@@ -30,12 +30,21 @@ reqwest = { version = "0.11", default-features = false, features = [
   "gzip",
 ] }
 risc0-binfmt = { workspace = true }
+risc0-build = { workspace = true }
+risc0-zkvm = { workspace = true, features = ["prove"] }
```

Comment:
So this will essentially break the client/server model because it means anyone that wants to install `cargo-risczero` will now have to build the prover, which will cause very long compile times.

Why does `cargo-risczero` now depend on the `prove` feature?

Reply:
Addressed in c75f8b613b940896a4ae5be82eedfbad9a999b7f by making it activated by the `experimental` flag. It is required because the test command uses the `Executor`.